client/src/components/BookList.js

```diff
@@ -22,6 +22,8 @@ function BookList({ newBooks, listName, deleteBook }) {
           key={newBook.id}
           onCardBtnClick={() => deleteBook(newBook.id)}
           title={newBook.volumeInfo.title}
+          handleMoreBtnClick={() => showDetail(newBook.id)}
+          buttonMoreName="Mehr über's Buch"
         />
       ))}
     </BookListWrapper>
```

Comment:
Your books are all ``li``elements. To be consistent the ``BookListWrapper`` element could be a ``ul``/ ``ol``. If that does not serve well, you could maybe think of changing the html element of the bookcard. Not sure if it is necessary, but it was a thought, that came to my mind.